app/helpers/config/aca_helper.rb

```diff
@@ -364,4 +364,12 @@ def aca_dental_plan_options
   def aca_default_dental_plan_option
     Settings.aca.default_dental_option_kind
   end
+
+  def ivl_osse_eligibility_is_enabled?(year = TimeKeeper.date_of_record.year)
+    if year >= 2021
```

Comment:
remove if else block and change condition to...

 EnrollRegistry.feature?("aca_ivl_osse_subsidy_#{year}")  && EnrollRegistry.feature_enabled?("aca_ivl_osse_subsidy_#{year}")

Reply:
this is resolved